Main.cpp

```diff
@@ -0,0 +1,133 @@
+/* 
+ @file      Main.cpp
+ @brief     creates geometry and materials to run a Monte Carlo simulation
+ @author    Luke Eure
+ @date      January 9 2016
+*/
+
+#include <iostream>
+#include <vector>
+#include <time.h>
+#include <math.h>
+#include <stdlib.h>
+
+#include "Material.h"
+#include "Boundaries.h"
+#include "Distributions.h"
+#include "Tally.h"
+#include "Mesh.h"
+#include "Monte_carlo.h"
+#include "Plotter.h"
+#include "Neutron.h"
+
+int main() {
+    
+    srand(time(NULL));
```

Comment:
This is not typically a good idea.  It means there is no reproducibility in the results (i.e., you can't run it twice and get the same answer).  This also makes it hard to debug because the behavior of the simulation changes every time you run it.

Instead: make this a set number, but make it possible to change that number in an input file or command-line argument